tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<ul><li>"...a gaussian profile whose standard deviation..."</li><li>Furthemore, the amplitude of the laser fluctuates according to a normal distribution with a standard deviation of  <code>amp_sigma</code> .</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>